tasks/deploy/lib/constants.ts

```diff
@@ -91,6 +91,32 @@ export const WSTETH_WHALE_MAINNET =
 export const SEPOLIA_USDE_ADDRESS =
     "0x9458CaACa74249AbBE9E964b3Ce155B98EC88EF2" as Address;
 
+// ╭─────────────────────────────────────────────────────────╮
+// │ Gnosis Addresses                                        │
+// ╰─────────────────────────────────────────────────────────╯
+
+export const CHAINLINK_AGGREGATOR_ADDRESS_GNOSIS =
+    "0x6dcF8CE1982Fc71E7128407c7c6Ce4B0C1722F55" as Address;
```

Comment:
Let's remove this. I have this constant in the test to make setting the storage variable easier, but we're never going to want to use this in prod.